tests/suites/spaces_ec2/util.sh

```diff
@@ -22,6 +22,17 @@ add_multi_nic_machine() {
 		--instance-id $(juju show-machine --format json | jq -r ".[\"machines\"] | .[\"${juju_machine_id}\"] | .[\"instance-id\"]")
 }
 
+# configure_multi_mic_netplan()
+#
+# Patch the netplan settings for the new interface, apply the new plan,
+# restart the machine agent and wait for juju to detect the new interface
+# before returning.
+configure_multi_nic_netplan() {
+	local juju_machine_id hotplug_iface
+	juju_machine_id=$1
+	hotplug_iface=$2
+}
+
```

Comment:
Not sure what's going on here. This function is already defined below

I suspect the correct diff for this file would be 0